hub/hub-api.yaml

```
@@ -1624,6 +1529,27 @@ components:
           items:
             $ref: "#/components/schemas/LimitUsage"
 
+    UserCreationRequest:
+      type: object
+      description: A model to create a new user.
+      required:
+        - email
+      properties:
+        email:
+          type: string
+        firstName:
+          type: string
+        lastName:
+          type: string
+        isAdmin:
```

```suggestion
        admin:
```
we have a boolean getter prefix "is", so not to have isIsAdmin we can do this :)

---

I wanted to change this one too, but sell and the starter library use this one too and I wanted to keep my changes in this PR as small as possible (given that its already quite big)